src/frontend/csrs-portal/src/app/api/api/account.service.ts

```diff
@@ -368,6 +368,7 @@ export class AccountService {
         let localVarCredential: string | undefined;
         // authentication (Bearer) required
         localVarCredential = this.configuration.lookupCredential('Bearer');
+        console.log('localVarCredential ' + localVarCredential);
```

Comment:
please do not leave console log statements in the front end.

Reply:
deleted console.log line